app/views/admin/_roles_admin.html.erb

```diff
@@ -0,0 +1,14 @@
+<% roles.each do |role| %>
+  <td>
+    <label class="permissions-checkbox-label">
+      <%= check_box_tag "#{user.id}[#{role.to_s}]", 'foo', user.has_cached_role?(role),
+                        disabled: (role == :developer and not user.has_cached_role?(role)),
+                        class: "permissions-checkbox", data: {:user_id => user.id, :role => role.to_s } %>
+      <% if role.to_sym == :core %>
```

Comment:
You can actually knock this bit out completely - we don't need pinned roles any more now that core isn't automatic.